internal/app/config_preference.go

```diff
@@ -81,6 +84,56 @@ func (a *app) loadPreferencesConfig() {
 	a.ConfigRegister(cfg)
 
 	window.SetFps(cfg.Prefs.Interface.Fps)
+
```

Comment:
Minor polish suggestion: could you move the code below into a separate function? You can declare the function in the same file and call it from the `load` method.